src/main/resources/templates/user/signupForm.html

```diff
@@ -1,19 +1,62 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Title</title>
-</head>
-<body>
-<form th:action="@{/users/signup}" method="post">
-    <p th:if="${error}" th:text="${error}"></p>
-    <label for="username">Username</label>
-    <input type="text" name="username" th:value="${userDTO?.username}" id="username"/>
-    <label for="email">Email</label>
-    <input type="text" name="email" th:value="${userDTO?.email}" id="email"/>
-    <label for="password">Password</label>
-    <input type="password" name="password" id="password"/>
-    <button type="submit">Submit</button>
-</form>
-</body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
+    <link rel="stylesheet" type="text/css" th:href="@{/styles/index.css}">
+    <link href="https://fonts.googleapis.com/css2?family=Andika:wght@400;700&family=Montserrat:wght@200;400;600&family=Righteous&family=Ubuntu:wght@400;500;700&display=swap"
+          rel="stylesheet">
+    <script src="https://code.jquery.com/jquery-3.6.4.min.js"></script>
+</head>
```

Comment:
The head is missing favicon and also the title should not be login.
Maybe try to use our predefined fragment.